src/util/tcyml.js

```diff
@@ -0,0 +1,12 @@
+const request = require('superagent');
+const yaml = require('js-yaml');
+
+const CENTRAL_RAW = 'https://hg.mozilla.org/mozilla-central/raw-file/default/';
+
+exports.getCentralTaskclusterYml = async () => {
```

Comment:
We don't have a hook per project, though -- only per level.  We could potentially have one per repo, but then we have a hook per repo, per actionPerm, and we are granting those individual trigger-hook scopes to users, so it's quite an explosion.  I don't see a great benefit, either -- it would just leave us running `tcadmin` on every uplift, if there was something important changing in `.taskcluster.yml`.

Reply:
I'm happy if you call it only with `mozilla-central` (although I think we also need to call it with `comm-central`), but I think it is worth making the utility general, even if the callers aren't).